config.example.json

```diff
@@ -1,4 +1,6 @@
 {
+  "outpath": "./output",
+  "namespaceBase": "/",
```

Comment:
Indeed! I should indeed update the readme ;)

Op 30 aug. 2016 10:22 a.m. schreef "Pieter Colpaert" <
notifications@github.com>:

> In config.example.json
> <https://github.com/mmlab/owlturtle2http/pull/1#discussion_r76749779>:
>
> > @@ -1,4 +1,6 @@
> >  {
> > +  "outpath": "./output",
> > +  "namespaceBase": "/",
>
> What's this? The base from which to start after the http://{hostname}?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mmlab/owlturtle2http/pull/1/files/6441d146909d0e6ce662636179a6761caca6c4c3#r76749779>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAozpTnpH2_YHUmvBHkaaEhNazNmXAMYks5qk-gtgaJpZM4JwO9t>
> .
>